docs/study-list.md

```diff
@@ -18,6 +18,7 @@ for more details and any associated publications.
 - [Suicidality length of stay](https://github.com/smart-on-fhir/cumulus-library-suicidality-los)
 
 ### 2024
+- [Data Metrics](https://github.com/smart-on-fhir/cumulus-library-data-metrics)
```

Comment:
Alphabetic inside the year, or still always append?

Reply:
oh god. i had been doing chronological (which just happened to be the same) but maybe alpha is better?

Reply:
I guess it matters how important the chronology is?

Reply:
Switched to pure chronological to keep with your intent